temel_bilgiler/javascript_temelleri/fundamentals-1.md

```diff
@@ -0,0 +1,118 @@
+
+* Bir değişken nasıl tanımlanır?
+* Bir değişkeni tanımlamanın üç farklı yolu nelerdir?
+* Hangi birini ne zaman kullanmalısınız?
+* Değişkenler için adlandırma kuralları nelerdir?
+* Operatörler, işlem yapılandırıcıları ve işlemler nelerdir?
+* Birleştirme nedir ve sayıları ve dizeleri birleştirirken ne olur?
+* JavaScript'teki farklı operatör türleri nelerdir?
+* == ve === arasındaki fark nedir?
+* Operatör öncelik değerleri nelerdir?
+* Artırma / azaltma operatörleri nelerdir?
+* Bunları öne eklemenin ve sonuna eklemenin farkı nedir?
+* Atama operatörleri nelerdir?
+* Unary Plus Operator nedir?
+
+### JavaScript Kodunu Nasıl Çalıştırılır?
+
+Fundamentals kursunun çoğunda yazacağımız tüm JavaScript, tarayıcı üzerinden çalıştırılacaktır. Fundamentals ve NodeJS yolu sonraki derslerinde JavaScript'i tarayıcı ortamının dışında nasıl çalıştıracağınızı gösterecektir. Bu derslerin dışında, şimdilik istisna belirtilmedikçe JavaScript'inizi tarayıcıda çalıştırmaya her zaman varsaymalısınız, aksi takdirde beklenmedik hatalarla karşılaşabilirsiniz.
+
+Başlamak için, JavaScript kodunu içinde barındıran bir HTML dosyası oluşturmanız yeterlidir. Bilgisayarınızda herhangi bir dosyaya temel HTML iskeletini yazın:
+
+~~~html
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Sayfa Başlığı</title>
+</head>
+<body>
+  <script>
+    // JavaScript'inizi buraya yazın!
+    console.log("Merhaba, Dünya!")
+  </script>
+</body>
+</html>
+~~~
+
+Bu dosyayı kaydedin ve bir web tarayıcısında açın (bunu yapmak için Visual Studio Code'daki ["Live Server"](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) özelliğini kullanabilirsiniz!) ve daha sonra <span id="access-devTools-console">boş web sayfasına sağ tıklayıp "İncele" veya "Öğeyi İncele" seçeneğini seçerek tarayıcının konsolunu açın. 'Geliştirici Araçları' panelinde 'Konsol' sekmesini bulun ve seçin</span>, burada `console.log` ifadesinin çıktısını görmelisiniz.
+
+> <span id="console-log">`console.log()` tarayıcınızın geliştirici konsoluna bir şey yazdırmak için kullanılan komuttur. Bu, bu ve gelecekteki tüm makalelerin ve egzersizlerin sonuçlarını konsoldan yazdırmak için kullanabilirsiniz.</span> Bu ve gelecekteki derslerdeki tüm örneklerle birlikte kod yazmanızı öneririz.
+
+Bir web sayfasına JavaScript dahil etmenin başka bir yolu, harici bir betiği içermektir. Bu, web sitenize harici CSS dokümanlarını bağlamakla çok benzerdir.
+
+~~~html
+  <script src="javascript.js"></script>
+~~~
+
+JavaScript dosyaları, stiller için `.css`'ye benzer şekilde `.js` uzantısına sahiptir. Harici JavaScript dosyaları, daha karmaşık betikler için kullanılır.
+
+### Değişkenler
+
+Değişkenleri, kodunuzdaki veriler için sadece "depolama konteynerleri" olarak düşünebilirsiniz. <span id="variable-declaration">Yakın zamana kadar, JavaScript'te bir değişken oluşturmanın tek yolu `var` ifadesiydi. Ancak, en yeni JavaScript sürümlerinde iki yol daha var: `let` ve `const`.</span>
```

Comment:
fix: ...~veriler için sadece "depolama konteynerleri"~...verilerin "saklama kutusu"...

typo: ~Ancak,~ Ancak -no comma-